src/OrbitGl/Track.cpp

```diff
@@ -82,6 +82,15 @@ void Track::DoDraw(PrimitiveAssembler& primitive_assembler, TextRenderer& text_r
   // Draw track's content background.
   Quad box = MakeBox(content_top_left, Vec2(GetWidth(), GetHeight()));
   primitive_assembler.AddBox(box, track_z, track_background_color, shared_from_this());
+
+  // Track header highlight on hover.
+  if (IsMouseOver()) {
+    const Color kOutlineColor = Color(128, 128, 128, 255);
```

Comment:
Can we make this `constexpr` or at `static const`? Otherwise, the naming should be snake_case.

Reply:
done